contracts/FleekERC721.sol

```diff
@@ -18,6 +18,10 @@ contract FleekERC721 is ERC721, FleekAccessControl {
     event NewTokenExternalURL(uint256 indexed token, string indexed externalURL, address indexed triggeredBy);
     event NewTokenENS(uint256 indexed token, string indexed ENS, address indexed triggeredBy);
 
+    event NewMirror(string indexed mirrorName, uint256 indexed tokenId, address indexed owner);
+    event NewMirrorScore(string indexed mirrorName, uint256 score, address indexed triggeredBy);
```

Comment:
Should we add tokenId to this event? Same with `RemovedMirror`

Reply:
My idea was have the unique identifier `Mirror Name` so the token id would not be needed here, but we can add it too.